src/components/Navbar.module.css

```diff
@@ -1,26 +1,27 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;600&family=Maven+Pro:wght@500&display=swap");
 
 .logo {
-  font-size: large;
+  /* font-size: large; */
   margin-top: 2px;
   color: blue;
 }
 
 .logoAndName {
   display: flex;
   align-items: center;
-  gap: 10px;
+  gap: 30px;
+  font-size: xx-large;
 }
 
 .navbar {
   display: flex;
   align-items: center;
+  justify-content: space-around;
   gap: 100px;
   background-color: black;
   color: white;
   height: 70px;
-  padding: 0 1%;
-  font: italic bolder x-large "Maven Pro", sans-serif;
+  font: bolder x-large "Maven Pro", sans-serif;
```

Comment:
Is "Maven Pro" used in other parts of the app? If so, I'd recommend making this a CSS variable. Then if it ever changes, you can modify it in 1 place.
If this is the only place, maybe it's fine as is.

Reply:
I using it all over the app. I imported the font from goole in App.css. But when I try to removed "Maven Pro", the shorthand malfunction

Reply:
I was thinking you can do this (below) but I'm not sure how you can make the variable global to all CSS files. That might be a challenge. It's fine as is.

```css
:root {
  --primary-font: "Maven Pro";
}
```

then
```
font: bolder x-large var(--primary-font), sans-serif;
```

See here: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties